docs/documentation/pages/DECKHOUSE-FAQ.md

```diff
@@ -251,24 +251,19 @@ To switch the Deckhouse cluster to using a third-party registry, follow these st
 
 ## How do I change the configuration of a cluster?
 
-The general cluster parameters are stored in the `ClusterConfiguration` structure. It contains parameters such as:
-
-- cluster domain: `clusterDomain`;
-- CRI used in the cluster: `defaultCRI`;
-- Kubernetes control plane version: `kubernetesVersion`;
-- cluster type (Static, Cloud): `clusterType`;
-- address space of the cluster's Pods: `podSubnetCIDR`;
-- address space of the cluster's services: `serviceSubnetCIDR` etc.
+The general cluster parameters are stored in the [ClusterConfiguration](installing/configuration.html#clusterconfiguration) structure. It contains parameters such as:
```

Comment:
Some cutted phrase here. `It contains parameters such as:`

Reply:
Fixed. Thanks.